aliskin/assignment1/anagram.py

```diff
@@ -0,0 +1,19 @@
+from collections import defaultdict
+
+def isAnagram(s, t, case_sensitive=True):
```

Comment:
Please use the official function naming style: lowercase words separated by underscores.
See https://www.python.org/dev/peps/pep-0008/#function-and-variable-names.